doc/releases/migration-guide-3.6.rst

```diff
@@ -450,6 +450,14 @@ Bluetooth
     :kconfig:option:`CONFIG_BT_MESH_SAR_RX_DISCARD_TIMEOUT`,
     :kconfig:option:`CONFIG_BT_MESH_SAR_RX_ACK_RETRANS_COUNT` Kconfig options.
 
+* Audio
+
+  * The BT_AUDIO_CODEC_LC3_* values from ``<zephyr/bluetooth/audio/lc3.h>`` have moved to
+    ``<zephyr/bluetooth/audio/audio.h>`` and have the ``LC3`` part of their names replaced by a
+    more semantically correct name: e.g.
+    ``BT_AUDIO_CODEC_LC3_CHAN_COUNT`` is now ``BT_AUDIO_CODEC_CAP_TYPE_CHAN_COUNT``,
+    ``BT_AUDIO_CODEC_LC3_FREQ`` is now ``BT_AUDIO_CODEC_CAP_TYPE_FREQ``, and
+    ``BT_AUDIO_CODEC_CONFIG_LC3_FREQ`` is now ``BT_AUDIO_CODEC_CFG_FREQ``, etc. (:github:`67024`)
```

Comment:
Some function and enum type names has also changed to used the more commonly used abbreviation of cfg and cap, e.g.:
bt_audio_codec_config_freq -> bt_audio_codec_cfg_freq
bt_audio_codec_capability_type -> bt_audio_codec_cap_type

Should you perhaps also mention this, as they also are part of public api?

Reply:
Sure, I added some of the enums here as well (but far from all of them).